app/controllers/clients_controller.rb

```diff
@@ -31,22 +31,12 @@ def update
     respond_to do |format|
       if @client.update(client_params)
         format.html { redirect_to @client, notice: 'Client was successfully updated.' }
-        format.json { render :show, status: :ok, location: @client }
       else
-        format.html { render :edit }
-        format.json { render json: @client.errors, status: :unprocessable_entity }
+        format.html { render :edit } unless @client.update(profile_params)
       end
     end
   end
 
-  def destroy
```

Comment:
at least I have written pundit restrictions and view conditionals for it, that I'll need to remove, in case this will be removed